youtube/guide-for-beginners.md

```diff
@@ -1,137 +1,96 @@
 
-[‎ ](LineBreak)
+**Looking for screenshots/more information about the ReVanced Extended patches? [Check out this repository!](https://github.com/ReVanced-Extended-Community/Patches-Documentation#patches-documentation)**
 
-### If you're looking for resources about how to patch on PC or other methods, please read the [**documentation**](https://github.com/inotia00/revanced-documentation).
-
-[ ](LineBreak)
-
-### Looking for screenshots/more information about the ReVanced Extended patches? [This repository is where you should look!](https://github.com/ReVanced-Extended-Community/Patches-Documentation)
-
-[ ](LineBreak)
-
-### *Last updated by /u/SpacellaryUS on 09/July/2023.*
+**Last updated by /u/SpacellaryUS on 09/July/2023.**
 
 *Please know that this guide is updated manually when I get the chance, so it may not be completely up-to-date right after a new patch has been released, but I try my best to do so as soon as possible.*
 
-‎[‎ ](LineBreak)
-
-# **For New Users:**
 
-### **Things To Know Before You Begin...**
+### **Things To Know Before You Begin**
 
 * Read the [Frequently Asked Questions](https://www.reddit.com/r/revancedextended/wiki/faq/).
 
-* Read the [In the Process of Updating - Wiki Guide](https://www.reddit.com/r/revancedextended/wiki/yt-guide/) for a comprehensive guide.
-
 * This guide is for patching YouTube.
 
-* Check Kobe's [RVX YT Music guide](https://www.reddit.com/r/revancedextended/comments/13ipnn0/yt_music_revanced_extended_guide/) for instructions on installing YT Music Extended.
-
 * This guide is for non-rooted devices.
 
-* This guide will use the ReVanced Manager method.
+* This guide will use the RVX Manager patching method. Check the [documentation](https://github.com/inotia00/revanced-documentation#revanced-extended-documentation) for info on the other patching methods.
 
-* Check the [documentation](https://github.com/inotia00/revanced-documentation) for the other methods/PC guide.
+* Check the [YT troubleshooting page](https://www.reddit.com/r/revancedextended/wiki/yt-troubleshooting/#wiki_issues_with_patching_.26amp.3B_installation) if you get stuck during this guide.
 
-[‎ ](LineBreak)
 
 # **1. Requirements:**
 
-* Your device must be running Android 8.0+
-
-* Your device must use the arm64-v8a architecture. You will be able to check what your device's architecture is at a later point in the tutorial. 
-
-#### **Downloading MicroG, ReVanced Manager, and the YouTube APK...**
+* Your device must be running Android 8.0 or newer.
 
-* Vanced MicroG `(latest version)`:
+* Your device must use the arm64-v8a architecture. (You will find out if it is in a moment) 
 
-**Download and install the Vanced MicroG APK from [here](https://github.com/TeamVanced/VancedMicroG/releases/latest). Or you can download an updated forked version from [inotia00's GitHub](https://github.com/inotia00/VancedMicroG/releases/latest).**

```

Comment:
made it plain text instead of bold bec the bold was too similar to the header of this section